packages/material-ui/src/Accordion/Accordion.test.js

```diff
@@ -26,101 +32,94 @@ describe('<Accordion />', () => {
   }));
 
   it('should render and not be controlled', () => {
-    const wrapper = mount(<Accordion>{minimalChildren}</Accordion>);
-    const root = wrapper.find(`.${classes.root}`).first();
-    expect(root.type()).to.equal(Paper);
-    expect(root.props().square).to.equal(false);
-    expect(root.hasClass(classes.expanded)).to.equal(false);
+    const { container } = render(<Accordion>{minimalChildren}</Accordion>);
+    const root = container.firstChild;
+    expect(root).to.not.have.class(classes.expanded);
   });
 
   it('should handle defaultExpanded prop', () => {
-    const wrapper = mount(<Accordion defaultExpanded>{minimalChildren}</Accordion>);
-    expect(findOutermostIntrinsic(wrapper).hasClass(classes.expanded)).to.equal(true);
+    const { container } = render(<Accordion defaultExpanded>{minimalChildren}</Accordion>);
+    expect(container.firstChild).to.have.class(classes.expanded);
   });
 
   it('should render the summary and collapse elements', () => {
-    const wrapper = mount(
+    const { getByText } = render(
       <Accordion>
         <AccordionSummary>Summary</AccordionSummary>
         <div id="panel-content">Hello</div>
       </Accordion>,
     );
-
-    expect(wrapper.find('[aria-expanded=false]').hostNodes().text()).to.equal('Summary');
-    expect(wrapper.find(Collapse).find('div#panel-content').text()).to.equal('Hello');
+    expect(getByText("Summary")).to.not.have.attribute("aria-expanded");
+    expect(getByText("Hello")).to.not.be.visible;
```

Comment:
```suggestion
    expect(getByText("Hello")).not.toBeVisible();
```

Reply:
🤔 Just tried this out, seems the `not` doesn't get applied properly to the function version of the assertion:
![image](https://user-images.githubusercontent.com/72429454/95649703-8a320d00-0b2a-11eb-9c96-12a56ecf8b34.png)

![image](https://user-images.githubusercontent.com/72429454/95649688-7c7c8780-0b2a-11eb-8ca0-00939bb60700.png)

Do you have ideas other than leaving it as-is? (which would mean I have to add an eslint ignore to it).